### ISSUE TITLE
Inggrit E Novaleta: Incremental Development, Week 1

### ISSUE THREAD

**Issue by inggritenovaleta** [HackYourFutureBelgium/class-13-14#85]
> [module repository](https://github.com/rago89/incremental-development)

### Incremental Development
<details>
<summary>expand/collapse</summary>
<br>

- [ ] [What is this?](https://www.youtube.com/watch?v=GzzkpAOxHXs)
- [ ] - [Agile Development by Cartoon](https://www.youtube.com/watch?v=Z9QbYZh1YXY&list=PLBUu5aGDLKnbeEx8U-5r436bw6p9wv1rS)
- [ ] - [Atlassian](https://www.atlassian.com/team-playbook/plays), [Team Playbook](https://www.atlassian.com/team-playbook/plays)
- [ ] - [Splitting User Stories](https://www.youtube.com/watch?v=EDT0HMtDwYI)
- [ ] - [Three key parts](https://www.youtube.com/watch?v=ctFzjMygaRo)
- [ ] - [`development-strategy.md` files](https://home.hackyourfuture.be/students/development-strategy)
- [ ] - [example development strategies](./development-strategies)
- [ ] - [example group repo](https://github.com/HackYourFutureBelgium/team-branchies)
- [ ] - All About Trees: [in steps](./all-about-trees-stepped), [as a repo](https://github.com/hackyourfuturebelgium/built-with-branches)

</details>

### Git & GitHub
<details>
<summary>expand/collapse</summary>
<br>

- [ ] - [learngitbranching](https://learngitbranching.js.org)
- [ ] - [git & github for poets](https://www.youtube.com/watch?v=BCQHnlnPusY&list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV)
- [ ] - lab.github.com:
- [ ]   - [First Day on GitHub](https://lab.github.com/githubtraining/first-day-on-github)
- [ ]   - [First Week on GitHub](https://lab.github.com/githubtraining/first-week-on-github)
- [ ] - [The Net Ninja](https://www.youtube.com/watch?v=QV0kVNvkMxc&list=PL4cUxeGkcC9goXbgTDQ0n_4TBzOO0ocPR&index=8)
- [ ] - [git-it](https://github.com/jlord/git-it-electron/)
- [ ] - [Understand how to use Atomic Commits](https://curiousprogrammer.io/blog/how-to-craft-your-changes-into-small-atomic-commits-using-git)

</details>

### Command Line Interface

<details>
<summary>expand/collapse</summary>
<br>

- [ ] - [Jesse Showalter](https://www.youtube.com/watch?v=5XgBd6rjuDQ)
- [ ] - CLI games:
- [ ]   - [bashcrawl](https://gitlab.com/slackermedia/bashcrawl/) - clone & play
- [ ]   - [Terminus](https://web.mit.edu/mprat/Public/web/Terminus/Web/main.html) - online
- [ ]   - [iTerm](https://sr6033.github.io/lterm/) - online
- [ ] - [A huge cheat sheet](https://gist.github.com/LeCoupa/122b12050f5fb267e75f)
- [ ] - [study.hyf.be](https://study.hackyourfuture.be/cli)

</details>

### Writing READMEs
<details>
<summary>expand/collapse</summary>
<br>

- [ ] - [makeareadme.com](https://www.makeareadme.com/)
- [ ] - [bulldogjob](https://bulldogjob.com/news/449-how-to-write-a-good-readme-for-your-github-project)
- [ ] - [meakaakka](https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3)

</details>

### DevTools and the DOM

<details>
<summary>expand/collapse</summary>
<br>

- [ ] - [How to inspect an element](https://www.lifewire.com/get-inspect-element-tool-for-browser-756549)
- [ ] - [inspecting-the-dom](https://hackyourfuturebelgium.github.io/inspecting-the-dom/)
- [ ] - [Modify the DOM. (does the source change?)](https://zapier.com/blog/inspect-element-tutorial/)

</details>

### HTML & CSS

<details>
<summary>expand/collapse</summary>
<br>

- [ ] - FCC, Responsive Web Design: [exercises](https://www.freecodecamp.org/learn), [video](https://www.youtube.com/watch?v=srvUrASNj0s)
- [ ] - [mmtuts: HTML & CSS](https://www.youtube.com/watch?v=TKYsuU86-DQ&list=PL0eyrZgxdwhwNC5ppZo_dYGVjerQY3xYU)
- [ ] - [CSS Games](https://study.hackyourfuture.be/html-css/css#games-to-learn-css)
- [ ] - :egg: [html-css-git-exercises](https://github.com/hackyourfuturebelgium/html-css-git-exercises)
- [ ] - :hatching_chick: [HTML-CSS-Practice-Problems](https://github.com/DevMountain/HTML-CSS-Practice-Problems)
- [ ] - :hatched_chick: [css-exercises](https://github.com/dangodev/css-exercises)

</details>

### Collaborating on GitHub

<details>
<summary>expand/collapse</summary>
<br>

- [ ] - [Adding collaborators to a repository](https://www.youtube.com/watch?v=p49LRx3hYI8)
- [ ] - [about code reviews](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews)
- [ ] - [requesting a code review](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review)
- [ ] - [Git Workflow for 2](https://github.com/hackyourfuturebelgium/git-workflow-workshop-for-two)
- [ ] - [Pull Requests](https://www.youtube.com/watch?v=2M16faxEQsg)
- [ ] - [Git & GitHub for Poets](https://www.youtube.com/watch?v=BCQHnlnPusY&list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV)
- [ ] - The Net Ninja: [11](https://www.youtube.com/watch?v=MnUd31TvBoU&list=PL4cUxeGkcC9goXbgTDQ0n_4TBzOO0ocPR&index=11)
- [ ] - linking PRs to Issues: [reference 1](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue), [reference 2](https://help.github.com/articles/autolinked-references-and-urls/)
- [ ] - [closing Issues using keywords](https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords)

</details>

### User Stories

<details>
<summary>expand/collapse</summary>
<br>

- [ ] - [User Stories](https://www.youtube.com/watch?v=apOvF9NVguA)
-  Acceptance Criteria
- [ ]   - [Yodiz](https://www.yodiz.com/blog/user-stories-acceptance-definition-and-criteria-in-agile-methodologies/)
- [ ]   - [Zepel](https://zepel.io/agile/acceptance-criteria-for-user-stories/)
- [ ]   - [Ruby Garage](https://rubygarage.org/blog/clear-acceptance-criteria-and-why-its-important)
- [ ]   - [The Infinity Project](https://www.youtube.com/watch?v=KYS0ptJ4JWc)
-  Setting Priorities
- [ ]   - [Many Strategies](https://zapier.com/blog/how-to-prioritize)
 - The Priority Quadrant
- [ ]     - [The Quadrant](https://www.youtube.com/watch?v=NGvsxPOmWuw)
- [ ]     - [Problems with this method](https://www.linkedin.com/pulse/why-prioritization-impacteffort-doesnt-work-itamar-gilad)
 - Other Perspectives
- [ ]     - [Alex Ponomarev](https://medium.com/swlh/prioritizing-user-stories-in-agile-projects-d1dd8dd79165)
- [ ]     - [Michael Lant](https://michaellant.com/2010/05/21/how-to-easily-prioritize-your-agile-stories/)

</details>

---

### Group Issue
    
- [x] [Group issue](https://github.com/HackYourFutureBelgium/class-13-14/issues/79)

**Comment by colevandersWands** [comment]
> branch - still learning to understand what the use of branch and how to do that

Perfect! This is exactly what you should be figuring out in this module.  Do you have any specific questions we can help you with?

**Comment by colevandersWands** [comment]
👋  @inggritenovaleta , can you add your week 2 check-in questions like in #89?  the `week-2` label is meant to let us know that you've filled in your check-in for this week.  It's hard to know how you're doing if you don't tell us!

**Comment by colevandersWands** [comment]
@inggritenovaleta , good idea starting early on the prep work.  This next Sunday will be a fun one
